rdtools/analysis.py

```diff
@@ -282,8 +282,10 @@ def pvwatts_norm(self, poa, cell_temperature):
                        "temperature_cell_ref": 25,
                        "gamma_pdc": self.temperature_coefficient}
 
+        # suppress RdTools 2.0 deprecation warning
+        warnings.filterwarnings("ignore")
         energy_normalized, insolation = normalization.normalize_with_pvwatts(self.pv_energy, pvwatts_kws)
-
+        warnings.resetwarnings()
```

Comment:
We should consider using non-deprecated functionality here instead of suppressing the warning -- we'll have to do it anyway in 3.0